packages/tests/server/mergeRouters.test.ts

```diff
@@ -17,6 +18,26 @@ test('mergeRouters', async () => {
   await expect(caller.bar()).resolves.toBe('bar');
 });
 
+test('merge routers with spread operator', async () => {
+  const t = initTRPC.create();
+
+  const router1 = {
+    foo: t.procedure.query(() => 'foo'),
+  } satisfies TRPCRouterRecord;
+  const router2 = {
+    bar: t.procedure.query(() => 'bar'),
+  } satisfies TRPCRouterRecord;
+
+  const merged = t.router({
+    ...router1,
+    ...router2,
+  });
```

Comment:
niceee - can we deprecate this monstosity then?

https://github.com/trpc/trpc/blob/next/packages/server/src/unstable-core-do-not-import/router.ts#L272-L309

or maybe just make it an alias?

Reply:
yes that'd be nice

Reply:
we could alias it